.ci_support/osx_arm64_mpiopenmpinumpy1.26python3.12.____cpython.yaml

```diff
@@ -9,7 +9,7 @@ channel_sources:
 channel_targets:
 - conda-forge main
 hdf5:
-- 1.14.2
+- 1.14.3
```

Comment:
did you manually change the pin? I'm mostly curious as to how the migration works.

Was rerendering enough to get this pin to take effect?

Reply:
I did nothing besides rerender and bump the build number.

Reply:
great. thanks. I was hoping this would be the behavior